LTHPasscodeViewController/LTHPasscodeViewController.h

```diff
@@ -314,6 +314,13 @@
  */
 - (void)showLockScreenWithAnimation:(BOOL)animated withLogout:(BOOL)hasLogout andLogoutTitle:(NSString*)logoutTitle;
 /**
+ @brief				Used for displaying the lock. The passcode view is added directly on the keyWindow.
+ @param superview   The superview where is to be presented, used to measure the center of the view.
+ @param hasLogout   Set to @c YES for a navBar with a Logout button, set to @c NO for no navBar.
+ @param logoutTitle The title of the Logout button.
```

Comment:
Awesome! Does the lock look okay in the following cases?

- fullscreen first (new frame is full size directly)
- fullscreen after it was displayed over the popup (new frame is bigger than previous)
- over the popup first (new frame is small size directly)
- over the popup after it was displayed as fullscreen (new frame is smaller than previous)